profiles/scalingo.json

```diff
@@ -1,17 +1,20 @@
 {
   "name": "Scalingo",
-  "revision": "2015-07-22",
-  "vendor_verified": "2015-04-17",
+  "revision": "2016-07-22",
+  "vendor_verified": "2015-02-23",
```

Comment:
`vendor_verified` means the last date an official member of the provider reviewed the profile.